### PR TITLE
Regex library update

### PR DESCRIPTION
**Pull request by smobregon**
- fix unescaped characters in `special chars spacing` and `special chars spans` regexes
- add regex called `format existing scripture tags` to Enhancements/Percival that captures existing scripture references and formats them with `data-ref`

### CODE REVIEW

File: source/process/regex-library.md [hunk 1]
```diff
@@ -1,83 +1,84 @@
...
 - **apparatus symbols**: Find apparatus symbols.<br>F: <code>(\[ℵ]|&#x(?:2135;|E(?:00\[021];|5(?:0\[45E6FA];|1\[034679];))))</code>
 
-- **check lang**: Find special `lang` characters<br>F: <code>&#60;span class=&#34;(\[^&#34;]+)&#34;&#62;(\[^A-Z]\[^<]&#42;\[āåâêëėèēîīôöòōûüū]\[^<]&#42;)&#60;/span&#62;</code>
+- **check lang**: Find special `lang` characters<br>F: <code>&#60;span class=&#34;(\[^&#34;]+)&#34;&#62;(\[^A-Z]\[^<]\*\[āåâêëėèēîīôöòōûüū]\[^<]\*)&#60;/span&#62;</code>
 
-- **extract lang**: Choose <mark>'Extract'</mark> to create a list of italicized words. Use this list to look for untagged lang or translit<br>F: <code>&#60;span class=&#34;(italic|i)&#34;&#62;(\[^<]&#42;)&#60;/span&#62;</code>
+- **extract lang**: Choose <mark>'Extract'</mark> to create a list of italicized words. Use this list to look for untagged lang or translit<br>F: <code>&#60;span class=&#34;(italic|i)&#34;&#62;(\[^<]\*)&#60;/span&#62;</code>
 
-- **ampersands**: replace ampersands<br>F: <code>(\[a-z]+\s&#42;)&(\s&#42;\[a-z]+)</code><br>R: <code>\1&#38;\2</code>
+- **ampersands**: replace ampersands<br>F: <code>(\[a-z]+\s\*)\*(\s\*\[a-z]+)</code><br>R: <code>\1&#38;\2</code>
```
Comment (flintsteel7) [inline source/process/regex-library.md:29]: Looks like this is going to change the regex from `([a-z]+\s*)&(\s*[a-z]+)` to `([a-z]+\s*)*(\s*[a-z]+)`
Is that intended?

File: source/process/regex-library.md [hunk 2]
```diff
@@ -87,56 +88,58 @@ title: RegEx Library
...
+- **tables to ABBR 1**: convert tables to abbreviation lists<br>F: <code>&#60;tr&#62;\s\*&#60;td&#62;(.\*?)&#60;/td&#62;\s\*&#60;td&#62;(.\*?)&#60;/td&#62;\s\*&#60;/tr&#62;</code><br>R: <code>&#60;dt epub:type=&#34;glossterm&#34;&#62;&#60;dfn&#62;\1&#60;/dfn&#62;&#60;/dt&#62;&#60;dd epub:type=&#34;glossdef&#34;&#62;\2&#60;/dd&#62;</code>
 
-- **tables to ABBR 2**: after running tables to ABBR 1 use this regex to format the lists new lines<br>F: <code>&#60;dfn&#62;(.&#42;?)&#60;/dfn&#62;&#60;/dt&#62;&#60;dd epub:type=&#34;glossdef&#34;&#62;(.&#42;?)&#60;/dd&#62;</code><br>R: <code>\n            &#60;dfn&#62;\1&#60;/dfn&#62;\n          &#60;/dt&#62;\n          &#60;dd epub:type=&#34;glossdef&#34;&#62;\2&#60;/dd&#62;</code>
+- **tables to ABBR 2**: after running tables to ABBR 1 use this regex to format the lists new lines<br>F: <code>&#60;dfn&#62;(.\*?)&#60;/dfn&#62;&#60;/dt&#62;&#60;dd epub:type=&#34;glossdef&#34;&#62;(.\*?)&#60;/dd&#62;</code><br>R: <code>\n &#60;dfn&#62;\1&#60;/dfn&#62;\n &#60;/dt&#62;\n &#60;dd epub:type=&#34;glossdef&#34;&#62;\2&#60;/dd&#62;</code>
```
Comment (flintsteel7) [inline source/process/regex-library.md:93]: The spaces for formatting have been removed... I wonder if that's intended.

File: source/process/regex-library.md (hunk 1 above)
Comment (smobregon) [reply source/process/regex-library.md]: That was not intended, I fixed it by putting an escaped ampersand in place of the asterisk.

File: source/process/regex-library.md (hunk 2 above)
Comment (smobregon) [reply source/process/regex-library.md]: I added the correct spaces back in but reviewing with live-server shows that not all the spaces are rendering properly. I will look into this more.

**Pull request merged**